.github/workflows/master-build.yml

```diff
@@ -14,4 +14,4 @@ jobs:
       tooling-version: 16
       target-build: true
       target-lint: true
-      target-unit-tests: true
+      target-tests: true
```

Comment:
This will breaks the pipelines as the "unit-test" parameter is required by the template